setup.py

```diff
@@ -39,12 +39,11 @@
     package_data={"": ["py.typed", "locale/*/LC_MESSAGES/*.mo"]},
     python_requires=">=3.6",
     install_requires=[
+        "pulp-glue>=0.17.0.dev,<0.18",
```

Comment:
The concern I have here is that we'll need a pulp-cli release when we release a new pulp-glue. That may be OK - I'm just trying to envision the release-process for the tooling as we go forward. Thoughts?

Reply:
And bump2version will take care of it, i hope...